src/action.py

```diff
@@ -1,30 +1,79 @@
 
 import glob
 import click
-import os
+import re
+
+from sys import exit
+from os import listdir
+from os.path import join, dirname, relpath, isfile, isdir
 from models.config import Config
 from collections import defaultdict
 from itertools import chain
 
+#TODO DRY up the file leader - only define '/var/lib/adminware/tools' once
 class ClickGlob:
-    def __glob_configs(namespace):
-        parts = ['/var/lib/adminware/tools',
-                 namespace, '*/config.yaml']
-        paths = glob.glob(os.path.join(*parts))
-        return list(map(lambda x: Config(x), paths))
+    def __glob_dirs(namespace):
+        parts = ['/var/lib/adminware/tools', namespace, '*']
+        paths = glob.glob(join(*parts))
```

Comment:
I don't think we need to use `glob` here anymore. It was originally required when trying to find directories containing a `config.yaml` file. However now each directory is searched recursively, this easily replaces `glob`.

Can `listdir` be used instead? It wouldn't return the absolute path, but there needs to be some refactoring around paths anyway.

Reply:
done here https://github.com/alces-software/adminware/pull/87/commits/0196f7d450de3834317902cf7dea047c48dbff99